.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - nightly
```

Comment:
@geggleto: We already discussed the minimum requirements. PHP 5.5 will be supported.